aeron-system-tests/src/test/java/io/aeron/archive/ReplicateRecordingTest.java

```diff
@@ -93,6 +95,7 @@ public void before()
                 .threadingMode(ThreadingMode.SHARED)
                 .errorHandler(Tests::onError)
                 .spiesSimulateConnection(true)
+                .timerIntervalNs(TIMER_INTERVAL_NS)
```

Comment:
Is this still needed?

Reply:
@mjpt777 This was exposing the bug. Without it tests were passing ok.